services/web/server/src/simcore_service_webserver/activity/config.py

```diff
@@ -0,0 +1,12 @@
+""" projects activity's configuration
```

Comment:
Please do not copy&paste wildly, make sense of it ... time to use cookie-cutter? :-)